src/engine/ChromaticFilter.cs

```diff
@@ -0,0 +1,21 @@
+using Godot;
+
+/// <summary>
+///   A chromatic abberation filter
+/// </summary>
+public class ChromaticFilter : TextureRect
+{
+    private static ChromaticFilter instance;
```

Comment:
It seems that you made this a singleton in the autoload style, but didn't add this to the autoloads? So this is now impossible to use.

Reply:
Removed this script file, it was a remnant from the old way I was loading the shader